wdl2cwl/WdlV1_1.py

```diff
@@ -131,7 +141,10 @@ def main(argv) -> None:
         for a in ast.task_variables:
             print("----WARNING: SKIPPING VARIABLE "+str(a[1])+"----")
 
+    with open('result.cwl', 'w') as result:
+        result.write(yaml.main.round_trip_dump(cat_tool.save()))
+        
```

Comment:
Maybe add another option to specify the output name?

Reply:
Or remove this section for now, as one can redirect the output to a file from the command line

Reply:
I kept it so that it's easy for me to see the resultant cwl output. Redirecting the output printed  'ANTLR runtime and generated code versions disagree: 4.9.2!=4.7.2' also to the file. Hence, why I took this approach.

Reply:
Okay, can you then add another command line option to specify the output filename? I would use the `argparse` module for this

Reply:
Ok I will look into that